cmake/modules/QtVersionAbstraction.cmake

```diff
@@ -13,6 +13,9 @@ if(UNIT_TESTING)
     find_package(Qt5Test REQUIRED)
 endif()
 
+find_package(Qt5WebEngineWidgets REQUIRED)
+find_package(Qt5WebEngine REQUIRED)
```

Comment:
Why are we not using find_package(Qt5 REQUIRED COMPONENTS WebEngineWidgets WebEngine) ?

Reply:
I must admit I just copied from above